.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseCJKNumberParser.cs

```diff
@@ -476,6 +476,8 @@ private double GetIntValue(string intStr)
 
             var isDozen = false;
             var isPair = false;
+            char roundNumberZero = '零';
```

Comment:
These values should not be hard coded here as the base parser is used for CJK. Any declarations like these two should be made in the YAML file and referenced in the parser config for each culture.